src/main/java/org/cyclonedx/maven/BaseCycloneDxMojo.java

```diff
@@ -176,30 +171,6 @@ public abstract class BaseCycloneDxMojo extends AbstractMojo implements Contextu
     @Parameter(property = "excludeTypes", required = false)
     private String[] excludeTypes;
 
-    /**
-     * Excluded ArtifactIds.
-     */
-    @Parameter(property = "excludeArtifactId", required = false)
-    protected String[] excludeArtifactId;
```

Comment:
This should continue to be in BaseCycloneDxMojo. Both makeBom and makeAggregateBom should account for exclusions.

Reply:
it currently is not taken into account in makeBom, or the refactoring would not have been ok: IIUC, this means there is a bug in makeBom, that will require another fix.
I'll revert that part of the refactoring in the current PR, and open another bugfix PR for taking the config into account into makeBom